### PR TITLE
spriting upgrades

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

replaces old roguetown sprites by their new versions made by 7sin

## Why It's Good For The Game

adds a visual improvement for the server and gives us a guide in how to make sprites from now on

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Review by hermaplusplus — CHANGES_REQUESTED** [review]
Multiple files have invalid file names. Please replace spaces in filenames with underscores or an otherwise appropriate name.

File: icons/roguetown/one handed weapons.dmi
Comment (hermaplusplus) [inline icons/roguetown/one handed weapons.dmi:1]: Invalid file name.

File: icons/roguetown/two handed weapon one hand.dmi
Comment (hermaplusplus) [inline icons/roguetown/two handed weapon one hand.dmi:1]: Invalid file name.

File: icons/roguetown/two handed weapons .dmi
Comment (hermaplusplus) [inline icons/roguetown/two handed weapons .dmi:1]: Invalid file name.

**Review by hermaplusplus — COMMENTED** [review]
Invalid filenames no longer present. Please fix the merge conflict.

**Review by hermaplusplus — CHANGES_REQUESTED** [review]
There is still a merge conflict.

**Review by hermaplusplus — APPROVED** [review]
Conflicts resolved

**Review by Dorodomki — COMMENTED** [review]
look good

**Pull request merged**